packages/server/lib/gui/links.ts

```diff
@@ -1,29 +1,6 @@
-import _ from 'lodash'
-import { URL, URLSearchParams } from 'url'
-
 // NOTE: in order for query params to be passed through on links
 // forwardQueryParams: true must be set for that slug in the on package
 
-interface OpenExternalOptions {
-  url: string
-  params: { [key: string]: string }
-}
-
-export const openExternal = (opts: OpenExternalOptions | string) => {
-  if (_.isString(opts)) {
-    return require('electron').shell.openExternal(opts)
-  }
-
-  const url = new URL(opts.url)
-
-  if (opts.params) {
-    // just add the utm_source here so we don't have to duplicate it on every link
-    if (_.find(opts.params, (_val, key) => _.includes(key, 'utm_'))) {
-      opts.params.utm_source = 'Test Runner'
-    }
-
-    url.search = new URLSearchParams(opts.params).toString()
-  }
-
-  return require('electron').shell.openExternal(url.href)
+export const openExternal = (url: string) => {
+  return require('electron').shell.openExternal(url)
```

Comment:
Any reason we inline the `require` instead of a top level `import`?

Reply:
That was changed as part of https://github.com/cypress-io/cypress/commit/cb5b547835ba5e58789b863589ced2212d2e8a13.

But good news, upgrading to electron 18 fixes this problem altogether and will allow that extra import to be removed. I'll get that taken out when https://github.com/cypress-io/cypress/pull/21418 lands and merges into 10.0-release.